### PR TITLE
update FindDepartmentFilter

### PR DESCRIPTION
**Pull request by vorobevamv**
in FindDepartmentFilter was removed property IncludeDeactivated, were added properties IsAscendingSort and IsActive;

in DepartmentRepository/FindAsync was removed filter for IncludeDeactivated; were added filters for IsActive and IsAscendingSort

### CODE REVIEW

File: src/DepartmentService.Data/DepartmentRepository.cs [hunk 1]
```diff
@@ -69,9 +69,14 @@ public async Task<(List<DbDepartment> dbDepartments, int totalCount)> FindAsync(
...
-      if (!filter.IncludeDeactivated)
+      if (filter.IsActive.HasValue)
       {
-        dbDepartments = dbDepartments.Where(d => d.IsActive);
+        dbDepartments = dbDepartments.Where(d => d.IsActive == filter.IsActive);
+      }
+
+      if (filter.IsAscendingSort.HasValue)
+      {
+        dbDepartments = filter.IsAscendingSort == true ? dbDepartments.OrderBy(d => d.Name) : dbDepartments.OrderByDescending(d => d.Name);
```
Comment (ndoriomedov) [inline src/DepartmentService.Data/DepartmentRepository.cs:79]: move ? to the 80th line, : to the 81st line

File: src/DepartmentService.Data/DepartmentRepository.cs [hunk 2]
```diff
@@ -69,9 +69,14 @@ public async Task<(List<DbDepartment> dbDepartments, int totalCount)> FindAsync(
     {
       IQueryable<DbDepartment> dbDepartments = _provider.Departments.AsQueryable();
 
-      if (!filter.IncludeDeactivated)
+      if (filter.IsActive.HasValue)
       {
-        dbDepartments = dbDepartments.Where(d => d.IsActive);
+        dbDepartments = dbDepartments.Where(d => d.IsActive == filter.IsActive);
```
Comment (MariyaSin) [inline src/DepartmentService.Data/DepartmentRepository.cs:74]: if IsActive ? return users where status IsActive == true : return users where IsActive == false

File: src/DepartmentService.Data/DepartmentRepository.cs (hunk 1 above)
Comment (MariyaSin) [reply src/DepartmentService.Data/DepartmentRepository.cs:79]: filter.IsActive.Value ? ... : ...

**Review by MariyaSin — APPROVED** [review]

File: src/DepartmentService.Data/DepartmentRepository.cs (hunk 1 above)
Comment (vorobevamv) [reply src/DepartmentService.Data/DepartmentRepository.cs]: done

**Review by actimellka — APPROVED** [review]

**Review by AnnPostnikova — APPROVED** [review]

**Review by EAndrey1091 — APPROVED** [review]

**Review by a5x55555 — APPROVED** [review]

**Pull request merged**